entity-management-web/src/test/java/eu/europeana/entitymanagement/validation/EntityFieldsValidatorTest.java

```diff
@@ -43,33 +45,54 @@
         Organization organization = objectMapper.readValue(loadFile(ORGANIZATION_VALIDATE_FIELDS_JSON), Organization.class);
-        EntityRecord entityRecord1 = new EntityRecord();
-        entityRecord1.setEntity(organization);
-        entityRecord1.setEntityId(organization.getEntityId());        
-        Set<ConstraintViolation<Entity>> violations = emValidatorFactory.getValidator().validate(entityRecord1.getEntity());
+        EntityRecord entityRecord = new EntityRecord();
+        entityRecord.setEntity(organization);
+        entityRecord.setEntityId(organization.getEntityId());        
+        Set<ConstraintViolation<Entity>> violations = emValidatorFactory.getValidator().validate(entityRecord.getEntity(), EntityFieldsCompleteValidatorGroup.class);
         for (ConstraintViolation<Entity> violation : violations) {
             System.out.println(violation.getMessageTemplate());
         }   
         //TODO: remove constraine violation: "The entity fields values are valid."
         Assertions.assertEquals(18, violations.size());
+        /*
+         * checking the minimal level of validation
+         */
+        violations.clear();
+        violations = emValidatorFactory.getValidator().validate(entityRecord.getEntity(), EntityFieldsMinimalValidatorGroup.class);
+        for (ConstraintViolation<Entity> violation : violations) {
+            System.out.println(violation.getMessageTemplate());
+        }   
+        //TODO: remove constraine violation: "The entity fields values are valid."
+        Assertions.assertEquals(22, violations.size());
     }
 
     @Test
-    public void validateEntityFieldsForAgent() throws JsonMappingException, JsonProcessingException, IOException {
+    public void validateEntityFieldsAgent() throws JsonMappingException, JsonProcessingException, IOException {
         
         Agent agent = objectMapper.readValue(loadFile(AGENT_VALIDATE_FIELDS_JSON), Agent.class);
-        EntityRecord entityRecord2 = new EntityRecord();
-        entityRecord2.setEntity(agent);
-        entityRecord2.setEntityId(agent.getEntityId());        
-        Set<ConstraintViolation<Entity>> violations = emValidatorFactory.getValidator().validate(entityRecord2.getEntity());
+        EntityRecord entityRecord = new EntityRecord();
+        entityRecord.setEntity(agent);
+        entityRecord.setEntityId(agent.getEntityId());        
+        Set<ConstraintViolation<Entity>> violations = emValidatorFactory.getValidator().validate(entityRecord.getEntity(), EntityFieldsCompleteValidatorGroup.class);
         for (ConstraintViolation<Entity> violation : violations) {
             System.out.println(violation.getMessageTemplate());
         }   
         //TODO: remove constraine violation: "The entity fields values are valid."
         Assertions.assertEquals(4, violations.size());
+        /*
+         * checking the minimal level of validation
+         */
+        violations.clear();
```

Comment:
the nimimal validation must be implemented in own integration test

Reply:
Done.